Include iproute2 in network e2e environment

mgarabed opened the pull request:
### What does this PR do?
Tests will fail in the e2e environment because the `ss` package isn't installed when we ask to `collect_connection_state`.

This PR includes it in our setup, plus adds a comment to the default YAML and README that the package will be required if that config option is set.

### Additional Notes
There seems to be a separate issue with the check not appropriately catching the `SubprocessOutputEmptyError` - but that gets avoided by having the package installed.

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] PR title must be written as a CHANGELOG entry [(see why)](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title)
- [ ] Files changes must correspond to the primary purpose of the PR as described in the title (small unrelated changes should have their own PR)
- [ ] PR must have `changelog/` and `integration/` labels attached

AlexandreYang commented on network/tests/common.py:
```diff
@@ -10,6 +10,10 @@
 
+
+# In order to collect connection state we need `ss` command included in `iproute2` package
+E2E_METADATA = {'start_commands': ['apt-get update', 'apt-get install iproute2 -y',]}
```
Looks like as expected, we now have more metrics. But the e2e was not failing before, so might be good to add some of the new metrics to `EXPECTED_METRICS` used for e2e:

Before 

```
    network (1.12.1)
    ----------------
      Instance ID: network:203b380dbea98c84 [OK]
      Configuration Source: file:/etc/datadog-agent/conf.d/network.d/network.yaml
      Total Runs: 2
      Metric Samples: Last Run: 37, Total: 74
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 15ms
```

Now

```
    network (1.12.1)
    ----------------
      Instance ID: network:203b380dbea98c84 [OK]
      Configuration Source: file:/etc/datadog-agent/conf.d/network.d/network.yaml
      Total Runs: 2
      Metric Samples: Last Run: 49, Total: 98
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 30ms
```

mgarabed replied:
Added additional metrics: https://github.com/DataDog/integrations-core/pull/5261/commits/b8f8a8c7e3331968632f72a410fcf47b4ac2045c

mgarabed commented:
Updated the README and config to specify that either `ss` or `netstat` could be used, but just stayed with `ss` for the E2E test.

Also updated the `pytest` module to better handle the JSON output of the E2E test.  This snippet overlaps with https://github.com/DataDog/integrations-core/pull/5204
